src/components/Outcome/OutcomeCategorical/TrendingOutcomeCategorical/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+import style from '../outcomeCategorical.scss'
+
+const cx = cn.bind(style)
+
+const TrendingOutcomeCategorical = ({
+  entryStyle, outcome, percentage, resolutionDate,
+}) => (
+  <div className="outcomes outcomes--categorical">
+    <div className={cx('outcomeWrapper')}>
+      <div className="entry__color" style={entryStyle} />
```

Comment:
Why not use `cx('entry')` here?

Reply:
@andre-meyer Will fix that, this is related to the question I asked you today about adding class names for elements with no style imported from .scss